### PR TITLE
Change smallBtn background color when hover to saveBtn color

### LINKED ISSUE #3535: Button highlighting is the wrong color
https://github.com/specify/specify7/assets/37256050/6d28f5c5-3217-4864-b582-86082eaf00f2

When hovering over an element, the button highlight color is now incorrect on `edge`.
Not a problem in `v7.8.11`.

### PR DESCRIPTION
**Pull request by CarolineDenis**
Fixes #3535

### CODE REVIEW

**Review by melton-jason — CHANGES_REQUESTED** [review]

**Comment by CarolineDenis** [comment]
@melton-jason @carlosmbe 
After discussion with @grantfitzsimmons we decided to unify the behavior for all the Button.Small. The background when hovering will be a darker grey in both modes. See videos bellow. 

https://github.com/specify/specify7/assets/108160931/dbf1aa12-5bdc-41fd-95dc-481e17130384




https://github.com/specify/specify7/assets/108160931/671abfa0-15b4-42ea-9c18-0b1c2a78721b

**Comment by melton-jason** [comment]
> After discussion with @grantfitzsimmons we decided to unify the behavior for all the Button.Small. The background when hovering will be a darker grey in both modes. 

That works for me, as long as there is a consensus :+1:

File: specifyweb/frontend/js_src/lib/components/Atoms/className.ts [hunk 1]
```diff
@@ -60,14 +60,14 @@ export const className = {
   secondaryButton,
   secondaryLightButton,
   niceButton,
-  smallButton: `${niceButton} !py-1 !px-2`,
+  smallButton: `${niceButton} !py-1 !px-2 hover:bg-[color:var(--save-button-color)]`,
```
[the remark this answers is not in the document]
Comment (melton-jason) [reply specifyweb/frontend/js_src/lib/components/Atoms/className.ts]: Resolved in 17d502a

> @melton-jason @carlosmbe After discussion with @grantfitzsimmons we decided to unify the behavior for all the Button.Small. The background when hovering will be a darker grey in both modes. See videos bellow.
> 
>  btns.mov 
>  Screen.Recording.2023-05-30.at.3.05.36.PM.mov

From https://github.com/specify/specify7/pull/3543#issuecomment-1569154335

**Review by carlosmbe — APPROVED** [review]
Looks Good 👍

**Review by melton-jason — APPROVED** [review]

**Comment by bronwyncombs** [comment]
`v7.8.12-prerelease`: Button.Small works as expected

**Comment by specifysoftware** [comment]
This pull request has been mentioned on **Specify Community Forum**. There might be relevant details there:

https://discourse.specifysoftware.org/t/specify-7-8-12-release-announcement/1202/1

**Pull request merged**